### PR TITLE
Use "pathParam" instead of "captureParam" to refer to path parameters

### LINKED ISSUE #344: The name captureParam is a bit strange
...because it doesn't describe what it is capturing. How about `pathParam`, cf.  https://swagger.io/docs/specification/describing-parameters/ ?

### PR DESCRIPTION
**Pull request by fumieval**
Fixes #344

### CODE REVIEW

File: Web/Scotty/Internal/Types.hs [hunk 1]
```diff
@@ -156,7 +156,7 @@ type Param = (Text, Text)
 type File = (Text, FileInfo LBS8.ByteString)
 
 data ActionEnv = Env { envReq       :: Request
-                     , envCaptureParams :: [Param]
+                     , envPathParams :: [Param]
```
Comment (ocramz) [inline Web/Scotty/Internal/Types.hs:159]: This is a breaking change (since the Types module is exposed) and should be marked as such.

**Review by ocramz — COMMENTED** [review]
Great! Could you add a changelog entry as well?

**Comment by ocramz** [comment]
@fumieval  the example programs in the README and in the Cabal file should be updated too then.

**Comment by fumieval** [comment]
>  the example programs in the README and in the Cabal file should be updated too then.
Right, fixed!

**Review by ocramz — APPROVED** [review]
👍

**Pull request merged**